mode/examples/Basics/Math/Map/Map.pyde

```diff
@@ -0,0 +1,27 @@
+'''
+ Map.
+ 
+ Use the map() function to take any number and scale it to a new number 
+ that is more useful for the project that you are working on. For example, use the
+ numbers from the mouse position to control the size or color of a shape. 
+ In this example, the mouse’s x-coordinate (a number between 0 and 360) is scaled to a
+ new number to define the color and size of a circle.
+ 
+ Example ReWritten in Python By: Prabhjot Singh (NITH)
+ example original in Java mode examples: Math->Maps
+'''
+def setup():
+    size(640, 360)
+    noStroke()
+
```

Comment:
Just 1 extra space between the end of `def setup(): ` and the start of `def draw():`.